cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client_test.go

```diff
@@ -203,6 +203,55 @@ func TestErrors(t *testing.T) {
 	mock.AssertExpectationsForObjects(t, server)
 }
 
+func TestMessageBasedErrors(t *testing.T) {
```

Comment:
That feels like it's duplicating a lot of TestErrors above. Wouldn't it be better to just errorMessage parameter to testCases config in TestErrors (and leave it empty for all other tests)?

Reply:
Updated.